### ISSUE TITLE
SeedsResolver relies on cluster_domain to be cluster.local

### ISSUE THREAD

**Issue by an-tex** [instaclustr/cassandra-operator#385]
As in https://github.com/instaclustr/cassandra-operator/blob/master/java/cassandra-k8s-addons/src/main/java/com/instaclustr/cassandra/k8s/SeedsResolver.java#L125

But the [cluster_domain](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/) can be configured differently by the cluster admin, in which case you get
```
│ java.lang.IllegalStateException: Unable to resolve any seeds!                                                                                              │
│     at com.instaclustr.cassandra.k8s.SeedProvider.getSeeds(SeedProvider.java:27) ~[cassandra-3-k8s-addons-1.0.3.jar:na]
```

and no cluster is formed. the actual `cluster_domain` can be read from [/etc/resolv.conf](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-dns-config), alternatively it should be possible to use `service.namespace` instead of `service.namespace.svc.cluster-domain.example` ?

**Comment by smiklosovic** [comment]
Hi @an-tex , thanks for reporting this, I ll check this soonish (e.g. tomorrow).

**Comment by smiklosovic** [comment]
@an-tex it should be fixed, please pull latest images and try it and let me know so I can do proper release

https://console.cloud.google.com/gcr/images/cassandra-operator/GLOBAL/cassandra-3.11.7?gcrImageListsize=30

Use tag `latest-dev`

**Comment by an-tex** [comment]
works like a charm! thanks a lot for handling it so swiftly and also merging my other PR, appreciate it :)

**Comment by smiklosovic** [comment]
@an-tex released as `v6.7.0`
